android/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.0'
+        classpath 'com.android.tools.build:gradle:7.4.2'
```

Comment:
Gradle 7.0 had a bug with multiple object files which we were running into. Updating to 7.4 seems to work. 7.4 was chosen randomly.